dspace/config/spring/oai/oai.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    The contents of this file are subject to the license and copyright
+    detailed in the LICENSE and NOTICE files at the root of the source
+    tree and available online at
+
+    http://www.dspace.org/license/
+
+-->
+<beans xmlns="http://www.springframework.org/schema/beans"
```

Comment:
Just a note here...since this Spring XML doesn't exist in 7.x, there would need to be a process to **automatically remove it** during the 6.x to 7.x upgrade.  Otherwise, it will cause errors when 7.x attempts to locate the beans defined here.  If I recall correctly, our current `ant update` process will only add in new Spring configs...it won't yet remove ones that don't exist anymore.

Reply:
if we agree on https://github.com/DSpace/DSpace/pull/2709#discussion_r393061152
no need to build a special remove during the upgrade

Reply:
This is fine, provided that it also is forward-ported to 7.0.  I was pointing out previously that this is a new feature not added to 7.0 yet.